api/swagger.yaml

```diff
@@ -5965,6 +5965,54 @@ paths:
           description: "Volume name or ID"
           type: "string"
       tags: ["Volume"]
+    put:
+      summary: "Update a volume"
+      operationId: "VolumeUpdate"
+      consumes: ["application/json"]
+      produces: ["application/json"]
+      responses:
+        200:
+          description: "The volume was updated successfully"
+          schema:
+            $ref: "#/definitions/Volume"
+        404:
+          description: "No such volume"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+        500:
+          description: "Server error"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
+      parameters:
+        - name: "name"
+          in: "path"
+          required: true
+          description: "Volume name or ID"
+          type: "string"
+        - name: "volumeConfigUpdate"
+          in: "body"
+          required: true
+          description: "Volume configuration update"
+          schema:
+            type: "object"
+            properties:
+              DriverOpts:
+                description: "A mapping of driver options and values to change. These options are passed directly to the driver and are driver specific."
+                type: "object"
+                additionalProperties:
+                  type: "string"
+              Labels:
```

Comment:
Aren't these the same - driverOpts and Labels, I'm guessing DirverOpts includes all of  the options provided as "-o<key=val> -o<key2=<val2>....". What is Labels used for?